src/core/a-entity.js

```diff
@@ -423,15 +425,15 @@ var proto = Object.create(ANode.prototype, {
   },
 
   removeComponent: {
-    value: function (name) {
+    value: function (name, force) {
```

Comment:
An example is that an entity without `position`, `rotation` or `scale` it does not make sense so those cannot be removed.